app/search_builders/hyrax/admin_set_search_builder.rb

```diff
@@ -8,6 +8,7 @@ class AdminSetSearchBuilder < ::SearchBuilder
     # @param [Symbol] access one of :edit, :read, or :deposit
     def initialize(context, access)
       @access = access
+      @rows = 100
```

Comment:
In the case I was addressing (/admin/admin_sets?locale=en) `#search_results_with_work_count?` isn't involved. The request comes in by way of: https://github.com/samvera/hyrax/blob/master/app/controllers/hyrax/admin/admin_sets_controller.rb#L36